app/src/app/[lng]/data/[step]/SourceDrawer.tsx

```diff
@@ -95,16 +95,15 @@ export function SourceDrawer({
               </Stack>
             </DrawerBody>
           )}
+          <Stack
+            w="full"
+            className="drop-shadow-top border-t-2 absolute left-0 flex justify-center items-center"
+          >
+            <Button onClick={onConnectClick} w="543px" h={16} my={6}>
```

Comment:
Do we need to hardcode the width or can we just add horizontal padding and make it depend on the text?

Helpful once we translate this page.